keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -190,8 +192,10 @@ spec:
   generateBucketName: [4]
   bucketClassName: [5]
   secretName: [6]
+  requestProtocol: [7]
+  requestAccessMode: {"ro", "rw"} [8]
```

Comment:
should this be plural, like PVCs?

Reply:
The accessModes here differ from PVCs pretty heavily.  The access modes with PVC are used to find a best match, which is a functionality we explicitly don't want to implement.  Best match is only useful for pre-provisioned, empty volumes.

Reply:
I agree with best match comment. My thinking now is that since we are treating access mode differently than PV-PVC and CSI then we should probably use a different field name.

Reply:
> Two comments re. BucketClass: 1) maybe we should add a `description` field. It could be in metadata or part of spec. 2) We should mention that, unlike storage classes, there is no default BC.

Description - will add, though it can't be part of metadata

Default BC - will add